app/models/temperature.py

```diff
@@ -0,0 +1,15 @@
+import datetime
+
+from sqlalchemy import Column, Integer, Float, ForeignKey, DateTime
+from sqlalchemy.orm import relationship
+
+from app.core.database import Base
+
+
+class Temperature(Base):
+    __tablename__ = "temperatures"
+    id = Column(Integer, primary_key=True, index=True)
+    city_id = Column(Integer, ForeignKey("cities.id"), nullable=False)
+    date_time = Column(DateTime, default=datetime.datetime.now(datetime.UTC))

```

Comment:
The `default` value for `date_time` uses `datetime.datetime.now(datetime.UTC)`, which is incorrect because `datetime.UTC` does not exist. You should use `datetime.datetime.utcnow()` or `datetime.datetime.now(datetime.timezone.utc)` to get the current UTC time.